packages/babel-generator/src/generators/types.js

```diff
@@ -216,19 +216,19 @@ export function StringLiteral(node: Object) {
 export function BigIntLiteral(node: Object) {
   const raw = this.getPossibleRaw(node);
   if (!this.format.minified && raw != null) {
-    this.token(raw);
+    this.word(raw);
```

Comment:
@nicolo-ribaudo @JLHwung we could also use `this.number` here if clearer (though there's logic in there for handling integer tokens), thoughts?

Reply:
Maybe let's add a comment explaining that we use `word` instead of `number` to avoid that additional logic?